common.ts

```diff
@@ -1,9 +1,18 @@
 import { loadAdaptableAppConfig } from "@adaptable/template";
 import { EnvSimple } from "@adpt/cloud";
 
+export interface ConfigDomain {
+    name?: string;
+    domainName: string;
+    endpoint?: string;
```

Comment:
Shouldn't this definition of endpoint match the one in the JSON schema?

Reply:
So you want endpoint to have type "main"?  I was doing enforcement of legal values in the schema, but here I would like the code to handle arbitrary endpoints correctly.  I can change it if you'd like though, I'm not firmly in one camp or the other.

Reply:
Am I misunderstanding the schema? It looks to me like the schema defines endpoint as an object with one property named endpoint.

Reply:
Yes it does and that is a bug.  Ok, let me fix that.

Reply:
Fixed.

Reply:
More precisely, I fixed the schema, but this is still a string vs. "main" as an enum in the schema, let me know if you would like this to just have type "main".